.github/workflows/pr-plugin-build.yml

```
@@ -0,0 +1,194 @@
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ## 🔧 PR Test Plugin Available
+            
+            A test plugin has been generated for this PR that includes the modified files.
+            
+            **Version:** `${{ steps.version.outputs.version }}`
+            **Build:** [View Workflow Run](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})
+            
+            ### 📥 Installation Instructions:
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)